API/src/microservices/Assessment-Service/index.js

```diff
@@ -1,10 +1,22 @@
+/* eslint-disable no-console */
+/* eslint-disable require-await */
+/* eslint-disable no-unused-vars */
+/* eslint-disable sort-keys */
```

Comment:
```suggestion
```

DO NOT do this.  eslint rules are there for a reason.  If you need to disable them temporarily, fine, but this isn't acceptable for a Pull Request.